engine/apps/mobile_app/tasks.py

```diff
@@ -595,10 +595,10 @@ def _should_notify_user_about_shift_swap_request(
     except MobileAppUserSettings.DoesNotExist:
         return False  # don't notify if the app is not configured
 
-    return (
-        mobile_app_user_settings.info_notifications_enabled  # info notifications must be enabled
```

Comment:
If I understand it ok, the idea is to flag the swap request as notified if a notification should have been sent while notifications are disabled, so when enabling we don't try to send it later, right?

Reply:
Yes, that's the idea